examples/resnet_imagenet/main.py

```diff
@@ -16,11 +16,11 @@
 from composer.loggers import ProgressBarLogger, WandBLogger
 from composer.optim import CosineAnnealingWithWarmupScheduler, DecoupledSGDW
 from composer.utils import dist, reproducibility
-from data import build_imagenet_dataspec
-from model import build_composer_resnet
 from omegaconf import OmegaConf
 
 from examples.common.config_utils import log_config
+from examples.resnet_imagenet.data import build_imagenet_dataspec
+from examples.resnet_imagenet.model import build_composer_resnet
```

Comment:
?

Reply:
i have no idea how this slipped through previously, but it was causing my pyright errors or something. all package imports should be fully qualified.